web-app/react-app/components/AuthPage/schemas/common.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+
+import * as yup from 'yup'
+import zxcvbn from 'zxcvbn'
+
+export const email = yup.string()
+    .trim()
+    .required('Email is required')
+    .email('Email must be a valid email address')
+
+export const password = yup.string()
+    .required('Password is required')
+
+export const passwordWithStrength = yup.string()
+    .required('Password is required')
+    .min(8, 'Password must be at least 8 characters long')
+    .test('is-strong', 'Please use a stronger password', (value) => zxcvbn(value).score > 1)
```

Comment:
Maybe return `zxcvbn(value).feedback.warning` - to give the user better feedback so they can improve their password. 
https://github.com/dropbox/zxcvbn